jbpm-services/jbpm-services-ejb/jbpm-services-ejb-timer/src/main/java/org/jbpm/services/ejb/timer/EJBTimerScheduler.java

```diff
@@ -204,6 +210,22 @@ public void internalSchedule(TimerJobInstance timerJobInstance) {
 	}
 
 
+    private Serializable removeTransientFields(Serializable info) {
+        // removing transient fields
```

Comment:
Wouldn't it be easier to just set these transient fields to null?

Reply:
It is fairly more complex do that. You cannot guarantee how deep is the  tree and it is error prone